src/database/migrations/20241205040900_DeleteParticipantStatusFromParticipants.ts

```diff
@@ -0,0 +1,13 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.alterTable('participants', (table) => {
+    table.dropColumn('status');
+  });
+}
+
+export async function down(knex: Knex): Promise<void> {
+  await knex.schema.alterTable('participants', (table) => {
+    table.enu('status', ['initialize', 'awaitingApproval', 'approved']);
```

Comment:
should i leave this as is which sets all values to null or should i set all rows to 'approved'

Reply:
done